arm/Microsoft.MachineLearningServices/workspaces/computes/deploy.bicep

```
@@ -0,0 +1,132 @@
+param resourceId string = ''
+
+@description('Required. Set the object type.')
+@allowed([
+  'AKS'
+  'AmlCompute'
+  'ComputeInstance'
+  'Databricks'
+  'DataFactory'
+  'DataLakeAnalytics'
+  'HDInsight'
+  'Kubernetes'
+  'SynapseSpark'
+  'VirtualMachine'
+])
+param computeType string
+
+@description('Optional. The properties of the compute. Will be ignored in case "resourceId" is set.')
+param properties object = {}
+
+@description('Optional. Enable telemetry via the Customer Usage Attribution ID (GUID).')
+param enableDefaultTelemetry bool = true
+
+// Identity
+@description('Optional. Enables system assigned managed identity on the resource.')
+param systemAssignedIdentity bool = false
+
+@description('Optional. The ID(s) to assign to the resource.')
+param userAssignedIdentities object = {}
+
+// ================//
+// Variables       //
+// ================//
+var identityType = systemAssignedIdentity ? (!empty(userAssignedIdentities) ? 'SystemAssigned,UserAssigned' : 'SystemAssigned') : (!empty(userAssignedIdentities) ? 'UserAssigned' : 'None')
+
+var identity = identityType != 'None' ? {
+  type: identityType
+  userAssignedIdentities: !empty(userAssignedIdentities) ? userAssignedIdentities : any(null)
+} : any(null)
+
+// ============================= //
+// Existing resources references //
+// ============================= //
+resource machineLearningWorkspace 'Microsoft.MachineLearningServices/workspaces@2021-04-01' existing = {
+  name: machineLearningWorkspaceName
+}
+
+// =========== //
+// Deployments //
+// =========== //
+resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (enableDefaultTelemetry) {
+  name: 'pid-47ed15a6-730a-4827-bcb4-0fd963ffbd82-${uniqueString(deployment().name)}'
+  properties: {
+    mode: 'Incremental'
+    template: {
+      '$schema': 'https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#'
+      contentVersion: '1.0.0.0'
+      resources: []
+    }
+  }
+}
+
+resource machineLearningWorkspaceCompute 'Microsoft.MachineLearningServices/workspaces/computes@2022-01-01-preview' = if (deployCompute == true) {
+  name: name
+  location: location
+  tags: tags
+  sku: {
+    name: sku
+    tier: sku
+  }
+  parent: machineLearningWorkspace
+  identity: identity
+  properties: union({
+    computeLocation: computeLocation
+    description: computeDescription
+    disableLocalAuth: disableLocalAuth
+    computeType: computeType
+  }, (!empty(resourceId) ? {
+    resourceId: resourceId
+  } : {
+    properties: properties
+  }))
+}
+
+// =========== //
+// Outputs     //
+// =========== //
+@description('The name of the compute.')
+output name string = machineLearningWorkspaceCompute.name
+
+@description('The resource ID of the compute.')
+output resourceId string = machineLearningWorkspaceCompute.id
+
+@description('The resource group the compute was deployed into.')
+output resourceGroupName string = resourceGroup().name
```

I'd suggest to add this output and regenerate the readme file
```suggestion
output resourceGroupName string = resourceGroup().name

@description('The principal ID of the system assigned identity.')
output systemAssignedPrincipalId string = systemAssignedIdentity && contains(machineLearningWorkspaceCompute.identity, 'principalId') ? machineLearningWorkspaceCompute.identity.principalId : ''
```

---

@eriqua Is updated.